std/file.d

```diff
@@ -4562,7 +4562,8 @@ private struct DirIteratorImpl
 
             static auto trustedFindFirstFileW(typeof(searchPattern) pattern, WIN32_FIND_DATAW* findinfo) @trusted
             {
-                return FindFirstFileW(pattern.tempCString!FSChar(), findinfo);
+                return FindFirstFileExW(pattern.tempCString!FSChar(), FINDEX_INFO_LEVELS.FindExInfoBasic, findinfo
```

Comment:
There is a comma missing here.

Perhaps try the patch on your PC first (also to get some benchmark numbers).

Reply:
copy-paste error, it ran ~25% faster

Reply:
25% is a *LOT*. Was it with a warm/cold cache in both runs?

Reply:
You can run test for yourself, those are the results I observed.

Reply:
I do not have access to a Windows machine at the moment.